actor-typed/src/main/scala/org/apache/pekko/actor/typed/internal/jfr/Events.scala

```diff
@@ -96,7 +96,7 @@ final class DeliveryProducerReceived(val producerId: String, val currentSeqNr: L
 @StackTrace(false)
 @Category(Array("Pekko", "Delivery", "ProducerController")) @Label(
   "Delivery ProducerController received demand request")
-final class DeliveryProducerReceivedRequest(val producerId: String, val requestedSeqNr: Long, confirmedSeqNr: Long)
+final class DeliveryProducerReceivedRequest(val producerId: String, val requestedSeqNr: Long, val confirmedSeqNr: Long)
```

Comment:
* now that the source has moved to main dir - more checks are done - and this field is treated as unused
* seems best to make it a val like all the other fields - exposing it to users